configs/cylinder3d/cylinder3d_4xb4_3x_semantickitti.py

```diff
@@ -0,0 +1,39 @@
+_base_ = [
+    '../_base_/datasets/semantickitti.py', '../_base_/models/cylinder3d.py',
```

Comment:
Do the `Backbone` and `Architectures` in README.md need to be updated after adding Cyclinder3d?